install-dev/upgrade/sql/1.7.0.0.sql

```diff
@@ -115,7 +115,8 @@ ALTER TABLE `PREFIX_lang` ADD `locale` varchar(5) COLLATE utf8_unicode_ci NOT NU
 CREATE TABLE `PREFIX_authorization_role` (
   `id_authorization_role` int(10) unsigned NOT NULL auto_increment,
   `slug` VARCHAR(255) NOT NULL,
-  PRIMARY KEY (`id_authorization_role`)
+  PRIMARY KEY (`id_authorization_role`),
+  UNIQUE KEY (`slug`)
```

Comment:
yup, it could make sense. The unique is not mandatory (I added it mainly for the upgrade), but better have it anyway